Android.mk

```diff
@@ -17,8 +17,12 @@ LOCAL_PATH:= $(call my-dir)
 TA_PATH:= $(LOCAL_PATH)
 include $(CLEAR_VARS)
 
+# Extra flags for SDP usage
+USE_TEE_SDP ?= false
```

Comment:
To be consistent with other TA's and also OP-TEE nomenclature we should enable this based on CFG_SECURE_DATA_PATH